README.md

```diff
@@ -54,14 +54,14 @@ Event listeners are tiggered by these calls:
 - `process.stderr.write(string)`
 - `process.exit(code)`
 
-The console functions `log`, `info`, `warn` and `error` get replaced and the
+The console functions `log` and `error` get replaced and the
 original implementation is exposed:
 
 - `console.log.original`
-- `console.info.original`
-- `console.warn.original`
```

Comment:
OK, I'll put that back.  My original issue report suggested leaving those channels free and based on your reply to that issue, I thought you had accepted that idea.  I was thinking from the browser point of view but, I have better understanding of your use case now (I didn't realise you were using it in mocaccino!)...